pkg/aws/ec2/api/eni_cleanup.go

```diff
@@ -140,12 +149,13 @@ func (e *ENICleaner) cleanUpAvailableENIs() {
 			}); tagIdx != -1 {
 				switch *networkInterface.TagSet[tagIdx].Value {
 				case config.NetworkInterfaceOwnerTagValue:
-					vpcrcLeakedENICleanupCnt.Inc()
+					vpcrcAvailableENICnt.Inc()
 				case config.NetworkInterfaceOwnerVPCCNITagValue:
-					vpcCniLeakedENICleanupCnt.Inc()
+					vpccniAvailableENICnt.Inc()
 				default:
-					// We will not hit this case as we only filter for above two tag values, adding it for any future use cases
-					e.Log.Info("found available ENI not created by VPC-CNI/VPC-RC")
+					// We should not hit this case as we only filter for relevant tag values, log error and continue if unexpected ENIs found
+					e.Log.Error(fmt.Errorf("found available ENI not created by VPC-CNI/VPC-RC"), "eni id", *networkInterface.NetworkInterfaceId)
```

Comment:
nit: Can you remove the space? Cloudwatch filter is not friendly for spaces on key words.